src/Worker.Extensions.DurableTask/HTTP/DurableHttpRequest.cs

```diff
@@ -19,20 +19,10 @@ public class DurableHttpRequest
     /// </summary>
     public DurableHttpRequest(
```

Comment:
nit: can inline params now
```suggestion
    public DurableHttpRequest(HttpMethod method, Uri uri)
```